tests/ui/fn/ok.stderr

```diff
@@ -0,0 +1 @@
+Error: failed to prove {@ wf((rigid &(shared) !lt_2 !ty_1))} given {}, got {}
```

Comment:
about which function is this error?

maybe create one test per failing function.